integration-tests/tests/delete-orphaned-workspaces.js

```diff
@@ -36,9 +36,10 @@ const deleteOrphanedWorkspaces = withUserToken(async ({ page, testUrl, token })
   const oldWorkspaces = await listOrphanWorkspaces(page);
   // List orphans which are already in state DeleteFailed (resistant to automated delete, don't fail on these)
   const oldWorkspaceNamesInDeleteFailed = getDeleteFailedWorkspaceNames(oldWorkspaces);
+  const oldWorkspacesToDelete = oldWorkspaces.filter(({ workspace: { name } }) => !oldWorkspaceNamesInDeleteFailed.includes(name));
```

Comment:
this is vulnerable if we ever create workspaces with the same name in different namespaces … which we don't do in the UI tests. Maybe just add a code comment for future engineers in case this ever happens?

Reply:
I'm not super worried about that, but I could certainly update this to include the namespace